test/app.test.js

```diff
@@ -1,391 +0,0 @@
-// Copyright IBM Corp. 2014,2016. All Rights Reserved.
```

Comment:
I think the workaround to make sure that this test is run last is poor. We should look to fix the bug that causes other tests to fail.

I remember having the same problem when I was investigating the test failures here. The hacky solution was to write `gen.emit('end')` after tests that involve error throwing. I remember you talking about a way to get the generator instance with the newest helper function, so I think this should be fixable.

Reply:
+1 for finding a more robust solution.